### PR TITLE
Added limited delta status at cli

### PR DESCRIPTION
**Pull request by ankit2001**
Screenshot from 2020-03-13 12-45-38
This pr is meant to solve issue #130 
please review ir @majurg

### CODE REVIEW

File: src/deltacode/cli.py [hunk 1]
```diff
@@ -55,6 +55,11 @@ def write_json(deltacode, outfile, all_delta_types=False):
     # TODO: add toggle for pretty printing
     simplejson.dump(results, outfile, iterable_as_array=True, indent=2)
     outfile.write('\n')
+    # Output delta_stats at cli
+    status=deltacode.stats.to_dict()
+    click.echo("\nDelta stats .....\n")
```
Comment (steven-esser) [inline src/deltacode/cli.py]: I would change `....` to `:`

File: src/deltacode/cli.py [hunk 2]
```diff
@@ -55,6 +55,11 @@ def write_json(deltacode, outfile, all_delta_types=False):
...
+    for stat, value in status.iteritems():
+        click.echo(str(stat) + " : " + str(value))
```
Comment (steven-esser) [inline src/deltacode/cli.py]: Do we really need to cast as `str` here?

**Review by steven-esser — CHANGES_REQUESTED** [review]

**Comment by ankit2001** [comment]
Screenshot from 2020-03-13 23-17-13
@majurg please review

File: src/deltacode/cli.py [hunk 3]
```diff
@@ -55,6 +55,12 @@ def write_json(deltacode, outfile, all_delta_types=False):
     # TODO: add toggle for pretty printing
     simplejson.dump(results, outfile, iterable_as_array=True, indent=2)
     outfile.write('\n')
+    # Output delta_stats at cli
+    status=deltacode.stats.to_dict()
+    click.echo("\nDelta stats :\n")
+    for stat, value in status.iteritems():
+        click.echo("{} : {}".format(stat,value))
+    #click.echo("\n")
```
Comment (steven-esser) [inline src/deltacode/cli.py]: Get rid of this commented out line if it is not needed.

File: src/deltacode/cli.py [hunk 4]
```diff
@@ -55,6 +55,12 @@ def write_json(deltacode, outfile, all_delta_types=False):
     # TODO: add toggle for pretty printing
     simplejson.dump(results, outfile, iterable_as_array=True, indent=2)
     outfile.write('\n')
+    # Output delta_stats at cli
+    status=deltacode.stats.to_dict()
```
Comment (steven-esser) [inline src/deltacode/cli.py]: Variable assignments should be separated by spaces.

**Review by steven-esser — CHANGES_REQUESTED** [review]
See inline comments. 

Also, you should improve your commit messages. A commit message of "Updates" tells me nothing of what has been done. See: https://chris.beams.io/posts/git-commit/

Please rebase your "updates" commits into a single commit and update the commit message to something more detailed using the link above.

**Comment by ankit2001** [comment]
I have rebased it into single commit , 
@majurg please review it.

**Pull request closed without merging**